x-pack/plugins/secops/public/components/flyout/index.tsx

```diff
@@ -108,30 +111,20 @@ export const FlyoutButton = pure(({ onOpen }: FlyoutButtonProps) => (
 ));
 
 export const FlyoutComponent = pure<Props>(({ show, timelineId, showTimeline, children }) => (
-  <React.Fragment>
-    <div
-      style={{
-        visibility: show ? 'visible' : 'hidden',
-      }}
-    >
-      <FlyoutPane
-        onClose={() => {
-          showTimeline(timelineId, false);
-        }}
-      >
+  <>
+    <Visible show={show}>
+      <FlyoutPane onClose={() => showTimeline({ id: timelineId, show: false })}>
         {children}
       </FlyoutPane>
       >
-    </div>
-    {!show ? (
+    </Visible>
+    {show ? null : (
```

Comment:
This is correct: we show the flyout button when `show` is false. My brain did a double-take though, so you could, (very optionally, this is not required for this cleanup PR, which is appreciated), extract the logic into a function that returns `true` if the flyout should be shown, so code above returns positive case first, and reads like:
```
showFlyoutButton(show) ?
  (<FlyoutButton
    timelineId={timelineId}	  
    onOpen={() => showTimeline({ id: timelineId, show: true })}
    />)
: 
  null
```

Reply:
I pushed the `show` into the Flyout and added two new unit tests. That should keep it very clean and consistent with other parts of the code.